src/main/external-resources/renderers/FetchTV.conf

```diff
@@ -0,0 +1,39 @@
+#----------------------------------------------------------------------------
+# Profile for Fetch TV (Mighty and Mini) @ https://www.fetchtv.com.au/
+# See DefaultRenderer.conf for descriptions of all the available options.
+#
+
+RendererName = Fetch TV
+RendererIcon = fetch-tv.png
+
+# ============================================================================
+# This renderer has sent the following string/s:
+#
+# User-Agent: Linux/3.14.28-1.6, UPnP/1.0, DLNADOC/1.50
+# ============================================================================
+#
+
+UserAgentSearch = Linux/3.14.28-1.6.*DLNADOC/1.50
```

Comment:
@Sami32 here it is. Thanks.
[ums_dbg.zip](https://github.com/UniversalMediaServer/UniversalMediaServer/files/1203223/ums_dbg.zip)